### PR TITLE
ISPN-8229 Add Max Content Length configuration to the REST endpoint

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.jboss.org/browse/ISPN-8229

### CODE REVIEW

File: persistence/rest/src/main/java/org/infinispan/persistence/rest/configuration/RestStoreConfigurationBuilder.java [hunk 1]
```diff
@@ -120,4 +127,6 @@ public void validate() {
          path(path + "/");
       }
    }
+
```
Comment (wburns) [inline persistence/rest/src/main/java/org/infinispan/persistence/rest/configuration/RestStoreConfigurationBuilder.java]: Extra whitespace

File: server/rest/src/main/java/org/infinispan/rest/configuration/RestServerConfigurationBuilder.java [hunk 2]
```diff
@@ -18,14 +18,16 @@
 
    public static final String DEFAULT_CONTEXT_PATH = "rest";
    public static final int DEFAULT_PORT = 8080;
-   public static final String DEFAILT_NAME = "rest";
+   public static final String DEFAULT_NAME = "rest";
+   public static final int DEFAULT_MAX_CONTENT_LENGTH = 5 * 1024 * 1024;
```
Comment (wburns) [inline server/rest/src/main/java/org/infinispan/rest/configuration/RestServerConfigurationBuilder.java]: Should this be 10M like above?

Comment (tristantarrant) [reply server/rest/src/main/java/org/infinispan/rest/configuration/RestServerConfigurationBuilder.java]: well spotted

**Comment by tristantarrant** [comment]
I have addressed your comments and also cleaned up the server cachestore side of things

**Comment by wburns** [comment]
Integrated into master, Thanks @tristantarrant !

**Pull request merged**